pages/content/pixi/index.md

```diff
@@ -25,6 +25,8 @@ staticText:
     tabsAriaLabel: Field and lab data tabs
     fieldData: Field data
     labData: Lab data
+    fieldDataExplainer: This data updated daily for the trailing 28-day period.
+    labDataExplainer: Please note that lab tests aren't reliable as they may vary per run.
```

Comment:
Not sure what the point being made here is?

Reply:
Taken from https://docs.google.com/document/d/15KTgFoQnmPEDhHyfANe-Ln5_qG20rTK1QOe4F6IfejQ/edit#
I believe the original intention was to caveat that lab data is a controlled/simulated test, especially if field data is not available.

However I'm all good with removing this entirely from the current proposed UI.

Reply:
We could also update to `Please note that lab tests may vary per run.` @nainar

Reply:
via Naina: We can also call out specifically that they are synthetic tests and do not impact page experience.